public/renderer-interface.proto

```diff
@@ -4,5 +4,6 @@ import public "decentraland/renderer/engine_interface.proto";
 import public "decentraland/renderer/renderer_services/crdt.proto";
 import public "decentraland/renderer/renderer_services/transport.proto";
 import public "decentraland/renderer/renderer_services/emotes_renderer.proto";
+import public "decentraland/renderer/renderer_services/rpc_scene_controller.proto";
```

Comment:
```suggestion
import public "decentraland/renderer/renderer_services/scene_controller.proto";
```

Can we change the name to `scene_controller`? Or maybe `remote_scene_controller`?

The rpc is another layer of the communication.

Reply:
yes, please find agreement with @pravusjif and @pbosio

Reply:
I think that being inside a path with name `renderer_services` already implies a lot, so `scene_controller` should be fine.
Unless we are planning for some other proto with a similar name, that I'm not aware of, and we might have a name collision?